src/hooks/PushNotification.ts

```diff
@@ -16,8 +16,15 @@ type ResponseHandler = (arg: Notifications.NotificationResponse) => void;
 export const usePushNotifications = (
   notificationHandler?: NotificationHandler,
   interactionHandler?: ResponseHandler,
-  behavior?: Notifications.NotificationBehavior
+  behavior?: Notifications.NotificationBehavior,
+  active?: boolean
 ): void => {
+  // this causes the active state to never change between rerenders.
+  // like this enabling or disabling the pushNotifications requires an app restart.
```

Comment:
we should add the information on activation/deactivation like for matomo: `...wird [...] mit dem nächsten Neustart der App wirksam.`

Reply:
this is happening when the SERVER settings get changed.
do you think there is a reasonable way to notify a user about that?

Reply:
aha, haven't read it in a way, that the server is meant here.